powa--5.0.2--5.0.3.sql

```diff
@@ -257,3 +257,17 @@ BEGIN
 END;
 $PROC$ LANGUAGE plpgsql
 SET search_path = pg_catalog; /* end of powa_stat_activity_src */
+DO
+LANGUAGE plpgsql
+$$
+DECLARE curr_table regclass;
+BEGIN
+  IF current_setting('server_version_num')::int >= 110000 THEN
+    FOR curr_table IN SELECT oid FROM pg_class WHERE EXISTS (SELECT 1 FROM pg_attribute WHERE attname = 'mins_in_range' AND pg_attribute.attrelid = pg_class.oid) AND 'toast_tuple_target=128' <> ALL(coalesce(reloptions,'{}'))
```

Comment:
I don't think that it's ok.  We can't assume that powa is installed on a database where no other table is matching this.

Also, this should probably be moved to a user visible function as otherwise users won't really be able to fix that easily if they pg_upgrade from pg10 and below to a pg11+ version.  Maybe the UI should eventually check that but we will definitely need the function to fix the situation.

There should also be a comment to mention that regclass::text cast is safe and cannot lead to SQL injection or any other problem.

Reply:
Yeah, I was thinking only about the collector scenario, sorry. Fixed I think